Views/MainWindow.xaml

```diff
@@ -7,103 +7,130 @@
         mc:Ignorable="d"
         Title="DiffChecker2" Height="200" Width="450"
         SizeToContent="Height">
-    <Grid>        
+    <Window.Resources>
+        <Style x:Key="style1" TargetType="{x:Type TextBlock}">
+            <Setter Property="Foreground" Value="Red"/>
+        </Style>
+        <Style x:Key="errorStyle"
+               TargetType="TextBlock">
+            <Setter Property="Validation.ErrorTemplate">
+                <Setter.Value>
+                    <ControlTemplate>
+                        <DockPanel LastChildFill="True">
+                            <TextBlock 
```

Comment:
pretty sure this has no effect? did not notice our text block having red foreground or text alignment "center"  or text wrapping